src/test/tests/rendering/ospray.py

```diff
@@ -140,7 +144,11 @@ def TestOsprayVolumeDomainBoundaries():
 
 def main():
     TestOsprayWindowSave()
-    TestOsprayVolumeDomainBoundaries()
+
+    #FIXME: this tests always has a slight pixel diff (~0.01%). We
+    # need to add the ability to set pixel diff tolerance for
+    # individual tests before we can run this in our nightly.
+    #TestOsprayVolumeDomainBoundaries()
```

Comment:
Would it be better to use the existing [*skip*](https://github.com/visit-dav/visit/blob/develop/src/test/skip.json) mechanism for this instead of commenting it out? That way there may be a little more awareness that we're skipping something important...

Reply:
Yeah, I can do that.

Reply:
@markcmiller86 I just need to add the case to the skip.json file, right?

Reply:
Yes, that is all think is needed. Then, we'll see `SKIPPED` in our routine testing and know we have something to revisit there at some point. Its too bad we cannot also include the *reason* we added the item to the skip list too but you've got the comment there and that seems good enough.